src/lesson_12/hometask_12/Student.java

```diff
@@ -0,0 +1,70 @@
+package lesson_12.hometask_12;
+
+import java.io.File;
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Student {
+    private String name;
+    ArrayList<File> fileList = new ArrayList<>();
+    private int score;
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public int getScore() {
+        return score;
+    }
+
+    public void setScore(int score) {
+        this.score = score;
+    }
+
+    public Student(String name, ArrayList<String> fileList, int score) {
+        this.name = name;
+        for (String fileName : fileList) {
+            try {
+                this.fileList.add(new File(fileName));
```

Comment:
you could add separate method addFile(String filename) or to use a setter for the files.